### ISSUE TITLE
loops

### ISSUE THREAD

**Issue by NickSeagull** [cloak-lang/cloak#19]

**Comment by MYF95** [comment]
Is ` |>` a special symbol or "keyword" we are adding? Although this question might not be related to the loop creation per se.

**Comment by NickSeagull** [comment]
Yes, it is a special operator called _pipe_, many functional languages implement it. It essentially does what `|` does in Bash:

* [Elm example](https://elm.christmas/2018/13)
* [Elixir example](https://elixirschool.com/en/lessons/basics/pipe-operator/)
* [Reason example](https://stackoverflow.com/questions/55474593/whats-the-difference-between-and-in-reasonml)
* [F# example](https://theburningmonk.com/2011/09/fsharp-pipe-forward-and-pipe-backward/)
* [Clojure has it too, but it is called "thread last"](https://riptutorial.com/clojure/example/29599/thread-last------)

See #29 for how our implementation would look like.
